lib/ansible/modules/packaging/os/redhat_repositories.py

```diff
@@ -0,0 +1,400 @@
+#!/usr/bin/python
+# vim:  et tw=120 ts=4 sw=4
+# Jerome Fenal (jfenal@redhat.com)
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+
+ANSIBLE_METADATA = {'status': ['preview'],
+                    'supported_by': 'community',
+                    'version': '1.0'}
+
+DOCUMENTATION = '''
+---
+module: redhat_repository
+short_description: Manage repositories with RHSM using the C(subscription-manager) command
+description:
+  - Manage repositories with the Red Hat Subscription Management entitlement platform using the C(subscription-manager) command
```

Comment:
Please wrap this line to the same line length as the rest of the documentation (I suggest 80 chars per line).